shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Answer.kt

```diff
@@ -1,8 +1,22 @@
 package co.nimblehq.ic.kmm.suv.domain.model
 
+interface Answerable {
+
+    val id: String
+    val content: String?
+    val placeholder: String?
+}
```

Comment:
Why do we need a new interface for Answer data class? 🤔

Reply:
Just want to try more with Kotlin's interface as I have less chance to work with it, no specific reason. The current approach still look good, right?

Reply:
@suho it's okay, just redundant 👍